examples/uefi-demo/main.rs

```diff
@@ -213,21 +248,83 @@ impl slint::platform::Platform for Platform {
         let info = gop.current_mode_info();
         let mut fb = alloc::vec![SlintBltPixel(BltPixel::new(0, 0, 0)); info.resolution().0 * info.resolution().1];
 
+        //mouse piexl
+        let mfb = alloc::vec![SlintBltPixel(BltPixel::new(255, 25, 25)); 20*20];
+
         self.window.set_size(slint::PhysicalSize::new(
             info.resolution().0.try_into().unwrap(),
             info.resolution().1.try_into().unwrap(),
         ));
 
+        let mut position = slint::LogicalPosition::new(0.0, 0.0);
+
+        let ptr = POINTER.load(Ordering::Relaxed);
+        let mpointer = unsafe { &mut *ptr };
+        let conpointer = unsafe { &*ptr };
+        let mouse_mode = conpointer.mode();
+
         loop {
             slint::platform::update_timers_and_animations();
 
-            while let Some(key) = get_key_press() {
```

Comment:
the previous code had a `while` there.  Why did you remove it. It allowed to process several key per frame if the user presses keys faster than we are able to render frames.

Reply:
> the previous code had a there. Why did you remove it. It allowed to process several key per frame if the user presses keys faster than we are able to render frames.`while`

thanks, now I know the code. I would wrap input with a loop later while next pr edit.